Message_chat/Extensions/UITableExtensions.swift

```diff
@@ -14,18 +14,8 @@ extension UITableView {
         }
         return cell
     }
-    func configureCell<T: UITableViewCell>(for indexPath: IndexPath,
-                                               with users: [User],
-                                               cellIdentifier: String,
-                                               cellType: T.Type) -> T {
-        let cell = self.dequeueReusableCell(withIdentifier: cellIdentifier,
-                                            for: indexPath) as! T
-        let user = users[indexPath.row]
-        if let cell = cell as? AcceptFriendsTableViewCell {
-            cell.setData(user: user)
-        } else if let cell = cell as? CancelFriendsTableViewCell {
-            cell.setData(user: user)
-        }
+    func configureCell<T: UITableViewCell>(for indexPath: IndexPath, cellIdentifier: String, cellType: T.Type) -> T {
+        let cell: T = self.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath) as! T
```

Comment:
viết kiểu này ko đúng r, khác gì chú bọc 1 hàm của ngta vào hàm của chú đâu, mà cũng ko ngắn hơn chút nào. Với ko đc dùng `!`, dễ gây crash. Tham khảo link dưới
https://stackoverflow.com/questions/55964080/register-and-dequeue-uitableviewcell-of-specific-type